shared/src/packages/orders/libs/validation-schemas/order-get-parameter.validation-schema.ts

```diff
@@ -0,0 +1,13 @@
+import joi from 'joi';
+
+import { type Id } from '~/libs/types/types.js';
+
+import { OrdersValidationMessage } from '../enums/enums.js';
+
+const orderGetParameter = joi.object<Id, true>({
+  id: joi.number().integer().positive().required().messages({
```

Comment:
There is a scheme in development to validate a positive id number, you can use it. It is located here: shared\src\libs\validation-schemas\validation-schemas.ts

Reply:
cool, fixed